__tests__/Unit/Components/Tasks/TasksContent.test.tsx

```diff
@@ -109,22 +87,20 @@ describe('tasks content', () => {
             </Provider>,
             { query: { section: 'available' } }
         );
-        await screen.findByTestId('tabs');
 
         const task = await findByText(
             'Design and develop an online booking system'
         );
         expect(task).toBeInTheDocument();
     });
 
-    test('display tasks when dev is true', async () => {
+    test('display tasks', async () => {
```

Comment:
can you please write proper test case message, 'display tasks' doesn't give me sense